extpsdk/build.gradle

```diff
@@ -11,7 +11,7 @@ ext {
         pypi_repo = 'testpypi'
     }
     if (!project.rootProject.hasProperty('vantiqConnectorSdkVersion')) {
-        vantiqConnectorSdkVersion = '1.3.0'
```

Comment:
took some tries to get it all worked out.

Intervening versions put only in test-pypi, so no interference.